lmdeploy/model.py

```diff
@@ -169,14 +169,15 @@ def messages2prompt(self, messages, sequence_start=True):
         return ret
 
 
+@MODELS.register_module(name='internlm-chat')
 @MODELS.register_module(name='internlm-chat-7b')
 class InternLMChat7B(BaseModel):
     """Chat template of InternLM model."""
 
     def __init__(self,
                  system='',
                  user='<|User|>',
-                 eoh='<eoh>',
+                 eoh='',
```

Comment:
So, we used the wrong prompt for internlm?

Reply:
Yes. They updated it